.github/workflows/setup-project.yaml

```diff
@@ -42,14 +42,18 @@ jobs:
     - name: Install Requirements
       shell: bash
       run: |
+        sudo apt-get update && sudo apt-get install --only-upgrade curl
+        sudo apt-get update && sudo apt-get install --only-upgrade make
+        sudo apt-get update && sudo apt-get install --only-upgrade python3       
         INSTALL_URL="https://raw.githubusercontent.com/sixfeetup/scaf/$GITHUB_REF_NAME/install.sh"
         export SCAF_SCRIPT_BRANCH=$GITHUB_REF_NAME
         git config --global init.defaultBranch main
         git config --global user.email "setup-project--install@example.com"
         git config --global user.name "CI CD install deps"
         echo "Downloading install script from $INSTALL_URL"
         rm -f $HOME/.local/bin/scaf
-        curl -sSL $INSTALL_URL | sh
+        echo "Installing scaf from $INSTALL_URL"
+        curl -sSL $INSTALL_URL | DEBUG=True bash
```

Comment:
GitHub actions are tripping up on this change since `scaf` is not being installed.